docs/content/doc/help/seek-help.en-us.md

```diff
@@ -18,6 +18,15 @@ menu:
 - [Discord](https://discord.gg/NsatcWJ)
```

Comment:
can you update this link to use the vanity URL?

Reply:
Done